bootloader/wscript

```diff
@@ -821,7 +851,7 @@ def build(ctx):
             source=['src/main.c'],
             target=exe_name,
             install_path=install_path,
-            use='OBJECTS USER32 COMCTL32 KERNEL32 ADVAPI32 GDI32 Z',
+            use='OBJECTS USER32 COMCTL32 KERNEL32 ADVAPI32 GDI32 Z STATIC_ZLIB',
```

Comment:
This `STATIC_ZLIB` target should be added to the else branch as well, otherwise we'll break the linux builds using `--static-zlib`. (If we're going to have and win-arm64 build check on the CI, perhaps we could also have one for linux with `--static-zlib`?).

Reply:
We might actually be able to drop the `--static-zlib` option for Linux. We only added it for #6006 to get around linking against `sstrip`-ed binaries on OpenWRT but now that we ship aarch64 musl wheels, no one should be building from source on OpenWRT anymore. But we can worry about that some other time. Sticking the check in CI sounds sensible to me.